demo/assets.js

```diff
@@ -1097,6 +1141,46 @@ shakaAssets.testAssets = [
       shakaAssets.Feature.MP4,
       shakaAssets.Feature.SEGMENT_TEMPLATE_DURATION
     ]
+  },
+  {
```

Comment:
Please enclose these assets in their own group.  For example, see line 1038 above, where we start a group for GPAC-sourced assets.